A1pr02_2

awanafiaz opened the pull request:
Incorporated all the changes discussed in the previous meeting, except the feedback form.

awanafiaz committed:
add_new_banner

New Banner Image.

awanafiaz committed:
relocate and add data files

Moved all data.csv files to data folder to limit cluttering the main folder. 
    1. Created a new Rmd file (dataFilesEditor_manuallyOperated) to make changes to the data files manually using R, keeping original data files the same. This will allows us to make changes to the table information according to the responses from the developers, should the need arise. 
    2. Added a column called “identifier” to the resource_table.csv to set as the key variable for merging with other tables.

awanafiaz committed:
added feedback placeholder

Added a new section “Feedback Form” for including the developer survey questionnaire.

awanafiaz committed:
merged links from resource, added descriptions, 

- Merged Links from Resource table to the general, imaging, omics, clinical tables (with minor aesthetic modification to column specifications)
    1. Additional package used: splitstackshape:: (library call included in the first chunk)
- Added draft description to each individual table.

awanafiaz committed:
changed site name, new homepage description

- Modified the homepage providing the description of the website and definitions of computing and data platforms
- Changed the website name to: ITCR Scientific Software Resources as instructed.
- Changed the Data Resources menu icon to: fa-database

awanafiaz committed:
added new exceptions

carriewright11 commented on dataFilesEditor_manuallyOperated.Rmd:
```diff
@@ -0,0 +1,46 @@
+resource_new <- resource %>% 
+  mutate(identifier = c("PDX", "HemOnc", "ngchm", "OncoMX", "MethCon5", "CIViC", "BioMuta", "TCIA", "SCGV", "ivyGlimpse", "TCPA", "dcmqi", "Region Templates", "Cistrome", "TOPAS", "TCGA", "1000 Genomes", "TARGET", "CCLE", "dbGAP", "CPTAC", "PheKB", "NSCLC Radiomics", "ICGC"))  %>%  relocate(identifier)
+
+write.csv(resource_new, file = "data/ictr_table_resources_new.csv")
```
I wonder if we could name this file something more informative than `_new` - maybe something about an identifier?

awanafiaz replied:
Agreed. I will rename it as "ictr_table_resources_identifier.csv" for better reference.

awanafiaz committed:
Updated merge routine

Updated merge routine for merging with resource table: removed dependency on new package.

awanafiaz commented:
List of changes included in this PR: 

1. Added a new banner image
2. Modified the homepage providing the description of the website and definitions of computing and data platforms
3. Moved all data.csv files to data folder to limit cluttering the main folder. 
    1. Created a new Rmd file (dataFilesEditor_manuallyOperated) to make changes to the data files manually using R, keeping original data files the same. This will allows us to make changes to the table information according to the responses from the developers, should the need arise. 
    2. Added (manually) a column called “identifier” to the resource_table.csv to set as the key variable for merging with other tables.
4. Merged Links from Resource table to the general, imaging, omics, clinical tables (with minor aesthetic modification to column specifications)
5. Added draft description to each individual table. 
6. Changed the website name to: ITCR Scientific Software Resources as instructed.
7. Changed the Data Resources menu icon to: fa-database
8. Added a new section “Feedback Form” for including the developer survey questionnaire.

carriewright11 commented on index.Rmd:
```diff
@@ -1,44 +1,37 @@
 ### **Welcome!**
 
-This website for ITCR Cancer Informatics Platform Database contains a comprehensive list of the bioinformatics platforms (computing and data) funded by **ITCR** *(Informatics Technology for Cancer Research)* grants. Each resource will have, along with their brief description, up to three links that navigate towards their -  
+This website for ITCR Scientific Software Resources contains a comprehensive list of the bioinformatics platforms (computing and data) that contain scientific software tools funded by **ITCR** *(Informatics Technology for Cancer Research)* grants.  
```
```suggestion
This website is intended to help cancer researchers find bioinformatics computing platforms and data resources, many of which are funded by the ITCR (Informatics Technology for Cancer Research) program of the NCI (National Cancer Institute). Please note that some of the information presented in these tables came from the [ITCR website](https://itcr.cancer.gov/informatics-tools).
```

carriewright11 replied:
@awanafiaz - you can commit suggestions from reviewers by pressing a button above if you would like to

carriewright11 replied:
note that if you do - you would want to use `git pull`  (or pull on GitKraken) to get the remote changes if you want to add more commits from your local machine

carriewright11 commented on index.Rmd:
```diff
@@ -1,44 +1,37 @@
+  - **Computing resource platforms** are web or cloud-based platforms that provide computational support to researchers including, but not limited to, analysis, visualization, storage, and computation of large-scale bioinformatics data sets while facilitating efficient workflow,  database management, wider collaboration,  reproducibility, and streamlined scientific communication.  
   
-Please browse the through the resources on each page by using the navigation bar above. 
+  - **Data resource platforms** are remotely hosted sources of curated or collaborative bioinformatics data that researchers can access and utilize as required. The available data sets accommodate genomics, proteomics, imaging (cancel and radiology), as well as other types of data.
```
```suggestion
- **Data resources** are remotely hosted sources of curated or collaborative bioinformatics data that researchers can access and utilize as required. The available data sets accommodate genomics, proteomics, imaging (histology and radiology), as well as other types of data.
```

carriewright11 commented on clinicalTable.Rmd:
```diff
@@ -5,12 +5,12 @@ output: html_document
 ### **Overview**
 
-This website contains a list of clinical informatics platforms funded by ITCR grants. Clinical platforms are those which work with several clinical data types, such as EMR Text and Clinical Notes. Each resource will have up to three links that navigate towards their main site, their documentation, and a list of their publications.
+The following table contains a list of clinical informatics platforms funded by ITCR grants. Clinical platforms work with several clinical data types, such as EMR Text and Clinical Notes, utilizing Natural Language Processing techniques, AI and machine learning algorithms in order to generate actionable insights with automated research tools. Note that, all resource summaries were taken from thei ...
```
```suggestion
The following table contains a list of clinical informatics platforms. These platforms work with several types of clinical data, such as EMR Text and Clinical Notes, utilizing Natural Language Processing techniques, AI and machine learning algorithms in order to generate actionable insights with automated research tools. 
```

carriewright11 commented on generalTable.Rmd:
```diff
@@ -3,14 +3,21 @@ title: "**General Informatics Table**"
 ### **Overview**
 
-This webpage contains a list of general informatics platforms funded by ITCR grants. General platforms are those which work with a wide variety of data types, from imaging data to “omics” data. Each resource will have up to three links that navigate towards their main site, their documentation, and a list of their publications.
+The following table contains a list of general informatics and biomedical platforms funded by ITCR grants. General platforms are those platforms which work with a wide variety of data types, from imaging data to “omics” data and provide computing resource support across different areas of bioinformatics research. Note that, all resource summaries were taken from their respective sources.
```
```suggestion
The following table contains a list of general informatics and biomedical platforms that can help researchers work with publicly available data and or their own data. General platforms are those which work with a wide variety of data types, from imaging data to “omics” data, and provide computing resource support across different areas of bioinformatics research. 
```

carriewright11 commented on imagingTable.Rmd:
```diff
@@ -5,12 +5,12 @@ output: html_document
 ### **About**
 
-This website contains a list of imaging informatics platforms funded by ITCR grants. Imaging platforms are those which work with several imaging data types, from pathology imaging to radiographic imaging. Each resource will have up to three links that navigate towards their main site, their documentation, and a list of their publications.
+The following table contains a list of imaging informatics platforms funded by ITCR grants. Imaging platforms work with several imaging data types, ranging from pathology imaging to radiographic imaging. These platforms allows researchers to create custom workflows, view and interact with analysis results, utilize imaging algorithms, analyze image tiles, as well as visualize and report analysis r ...
```
```suggestion
The following table contains a list of imaging informatics platforms. These platforms work with several imaging data types, ranging from pathology imaging to radiographic imaging. These platforms allow researchers to create custom workflows, view and interact with analysis results, utilize imaging algorithms, as well as visualize and report analysis results. 
```

carriewright11 commented on omicsTable.Rmd:
```diff
@@ -5,12 +5,12 @@ output: html_document
 ### **Overview**
 
-This website contains a list of "omics" informatics platforms funded by ITCR grants. "Omics" platforms are those which work with several genomic or proteomic data types, such as sequence data. Each resource will have up to three links that navigate towards their main site, their documentation, and a list of their publications.
+The following table contains a list of "omics" informatics platforms funded by ITCR grants. "Omics" platforms are those which work with several genomic or proteomic data types, such as sequencing, imaging, text and microarray data. These platforms allow access to interactively exploring and analyzing large-scale cancer data, create and share reproducible workspaces and results. The goal of these  ...
```
```suggestion
The following table contains a list of "omics" informatics platforms funded by ITCR grants. "Omics" platforms are those which work with genomic or proteomic data types, such as RNA-Seq data. These platforms allow access to interactively explore and analyze large-scale cancer data, as well as create and share reproducible workspaces and results. The goals of these platforms are to facilitate scientific collaboration and to better empower the research community for "omic" research. 
```

carriewright11 commented on resourceTable.Rmd:
```diff
@@ -5,12 +5,12 @@ output: html_document
 ### **Overview**
 
-This website contains a list of informatics resources funded by ITCR grants. Resources are remotely hosted sources of curated or collaborative data that researchers can use for projects. These resources also provide data to specific platforms highlighted within the various Platforms informatics tables.
+The following table contains a list of informatics resources funded by ITCR grants. Resources are remotely hosted sources of curated or collaborative data that are available to researchers. These platforms catalog common human genetic sequencing data, including genomic, epigenomic, transcriptomic, and proteomic data, cancer imaging data, radiology data, along with analytic and graphical tools to  ...
```
```suggestion 
The following table contains a list of cancer data resources that are available to researchers. These resources include genomic, epigenomic, transcriptomic, proteomic, and cancer imaging data. 
```

carriewright11 commented on dataFilesEditor_manuallyOperated.Rmd:
```diff
@@ -0,0 +1,45 @@
+
+### **Overview**
+
+Manually run the following r-chunks to make permanent edits to the data files into datafile_new.csv 
```
```suggestion
```

carriewright11 replied:
don't think we need this

carriewright11 commented on dataFilesEditor_manuallyOperated.Rmd:
```diff
@@ -0,0 +1,45 @@
+
+Manually run the following r-chunks to make permanent edits to the data files into datafile_new.csv 
+
+## DO NOT knit this Markdown file.
```
```suggestion
```

carriewright11 replied:
also don't need this

carriewright11 replied:
since we are creating a new file and have version control it's ok

carriewright11 reviewed (approved):
I have some changes to the descriptions and I think we can remove the information about manually running the code to add the identifier based on what we talked about in our meeting. Looks good to me if you make the suggested commits!

awanafiaz committed:
Update index.Rmd

Co-authored-by: Carrie Wright <23014755+carriewright11@users.noreply.github.com>

awanafiaz committed:
Update index.Rmd

Co-authored-by: Carrie Wright <23014755+carriewright11@users.noreply.github.com>

awanafiaz committed:
Update clinicalTable.Rmd

Co-authored-by: Carrie Wright <23014755+carriewright11@users.noreply.github.com>

awanafiaz committed:
Update generalTable.Rmd

Co-authored-by: Carrie Wright <23014755+carriewright11@users.noreply.github.com>

awanafiaz committed:
Update imagingTable.Rmd

Co-authored-by: Carrie Wright <23014755+carriewright11@users.noreply.github.com>

awanafiaz committed:
Update omicsTable.Rmd

Co-authored-by: Carrie Wright <23014755+carriewright11@users.noreply.github.com>

awanafiaz committed:
Update resourceTable.Rmd

Co-authored-by: Carrie Wright <23014755+carriewright11@users.noreply.github.com>

awanafiaz committed:
Update dataFilesEditor_manuallyOperated.Rmd

Co-authored-by: Carrie Wright <23014755+carriewright11@users.noreply.github.com>

awanafiaz committed:
Update dataFilesEditor_manuallyOperated.Rmd

Co-authored-by: Carrie Wright <23014755+carriewright11@users.noreply.github.com>

awanafiaz committed:
PR comments fix, merge code update, removed warnings

Incorporated all PR comments and changes discussed in meeting.

awanafiaz committed:
renamed resource file

added _identifier in name

awanafiaz committed:
updated editor 

removed redundancy
